patchwork-capabilities/src/main/java/net/minecraftforge/event/AttachCapabilitiesEvent.java

```diff
@@ -49,32 +49,33 @@ public AttachCapabilitiesEvent() {
 		this(null, null);
 	}
 
-	public AttachCapabilitiesEvent(Class<T> type, T obj) {
+	public AttachCapabilitiesEvent(Class<T> type, T object) {
 		super(type);
-		this.obj = obj;
+		this.object = object;
 	}
 
 	/**
-	 * Retrieves the object that is being created, Not much state is set.
+	 * Retrieves the object that is being created.<br>
+	 * <b>Note:</b> Object creation is still incomplete at this point
 	 */
 	public T getObject() {
-		return this.obj;
+		return this.object;
 	}
 
 	/**
-	 * Adds a capability to be attached to this object.
+	 * Adds a {@link net.minecraftforge.common.capabilities.Capability} to be attached to this object.
```

Comment:
After review it looks like capability provider is the more accurate name...
```suggestion
	 * Adds a {@link ICapabilityProvider capability provider} to be attached to this object.
```